src/components/grid-aware/HomePageLargeParagraph/HomePageLargeParagraph.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import s from "./HomePageLargeParagraph.module.css";
+
+const HomePageLargeParagraph = ({ title, description }) => {
+  return (
+    <div className={s.bleedWrapper}>
+      <div className={s.bleedMainContent}>
+        <div className={s.gridParent}>
+          <div className={s.textWrapper}>
+            <p className={s.title}>{title}</p>
```

Comment:
p tags should turn into div elements. There may be inherent properties in p that may need to be resetted. If this is implemented for semantic reasons, a p element should not have a title-like behavior.

Reply:
👍 Changed p tag to div.